### PR TITLE
David-Dev

### PR DESCRIPTION
**Pull request by dhurley-suncor**
- added new information on CPT popup labels
- added csv folder for raw data prior to json format
- added python scripts to process csv data to json and convert Mine --> utm --> lat/lon coords for SteepBank

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -222,7 +226,26 @@ function bubbleClicked(e) {
 
         let html = ['<div style="padding:10px;">'];
 
-        html.push(`<b>Dozer Pings: ${cluster.properties.dozerPings}</b><br/><br/>`);
```
Comment (dhurley-suncor) [inline scripts/main.js:225]: formatted html to be a bit more readable - could we create a template folder and import from there or maybe that become cumbersome with the variable reference?

Comment (ahasan-suncor) [reply scripts/main.js:225]: yeah we can look into that

File: data/queries/get_all_cell_boundary.sql [hunk 2]
```diff
@@ -0,0 +1,220 @@
+/*
```
[the remark this answers is not in the document]
Comment (ahasan-suncor) [reply data/queries/get_all_cell_boundary.sql:1]: i'll let you know

**Review by ahasan-suncor — APPROVED** [review]
Looks good!

**Pull request merged**